demo/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@
   SOFTWARE.
 ]]
 
+set(CMAKE_AUTOMOC true)
+cmake_minimum_required(VERSION 3.24)
```

Comment:
Why is this necessary? The top level CMakeLists file already specified the minimum required version

Reply:
Cmake didn't pick the top level CMakeLists for some reason, reverting the change